content/3_Silver/Graph_Traversal.mdx

```diff
@@ -619,7 +619,60 @@ See [here.](https://codeforces.com/blog/entry/67883)
 
 ### BFS Solution
 
-<IncompleteSection />
+<CPPSection>
+
+```cpp
+#include <cstdio>
+#include <queue>
+#include <vector>
+
+const int MN = 1e5 + 10;
+
+int N, M, ans, rep[MN];
```

Comment:
declarations on separate lines

Reply:
I have tried to make the bfs solution similar to the dfs one already uploaded on the silver section in graph traversal. You can check that in the silver section (Graph traversal).

Reply:
> i'll change that code later

that code itself was kinda bad

Reply:
ok then I will change the bfs code.